### PR TITLE
“it-localization-ingress-#2711”

### PR DESCRIPTION
**Pull request by matteobisi**
### Describe your changes
Ingress italian localization, new term

### Related issue number or link (ex: `resolves #issue-number`)
#2711 

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/it/ingress.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+La risorsa ingress è un file di configurazione e consente agli amministratori di configurare il routing del traffico esterno,
+l'ingress controller è un server web che esegue effettivamente il routing del traffico in base alla configurazione presente nella risorsa Ingress.
+
+## Quali problematiche affronta
+
+Le applicazioni web in ambienti cloud native, sono composte da diversi servizi e spesso è necessario che questi [servizi](/it/service/) siano accessibili via Internet per consentire agli utenti di visitarli tramite il loro browser.
+Per rendere accessibili questi servizi dell'applicazione agli utenti in ambiente [Kubernetes](/it/kubernetes/), è necessario esporre ciascun servizio dell'applicazione al mondo esterno.
+Il modo più diretto sarebbe utilizzare un servizio di bilanciamento del carico in Kubernetes.
+La creazione di un tale servizio comporta la generazione di un nuovo componente nell'infrastruttura sottostante;
+questo introduce nuovi costi e oneri gestionali, ma ogni nuovo bilanciatore di carico creato ha il proprio indirizzo IP esterno.
```
Comment (ugho16) [inline content/it/ingress.md:19]:
```suggestion
questo non solo introdurrebbe nuovi costi e oneri gestionali, ma ogni nuovo bilanciatore di carico creato avrebbe il proprio indirizzo IP esterno.
```

File: content/it/ingress.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+Questo porta a un'esperienza utente negativa, poiché, come utenti, non vogliamo navigare su URL differenti per accedere a un'applicazione
```
Comment (ugho16) [inline content/it/ingress.md:20]:
```suggestion
Questo porterebbe a un'esperienza utente negativa, poiché, come utenti, non vogliamo navigare su URL differenti per accedere a un'applicazione
```

File: content/it/ingress.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+## In che modo aiuta
+
+Una risorsa Ingress consente di configurare come il traffico viene bilanciato e instradato verso i servizi di un'applicazione.
```
Comment (ugho16) [inline content/it/ingress.md:24]:
```suggestion
Una risorsa ingress consente di configurare come il traffico viene bilanciato e instradato verso i servizi di un'applicazione.
```

File: content/it/ingress.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+L'ingress controller è un server web, eseguito all'interno del cluster, che interpretando le regole definite nella risorsa ingress  
+consente un singolo punto di ingresso attraverso un URL (www.example-url.com) e gestisce l'instradamento effettivo e il bilanciamento in base a diversi percorsi URL (www.example-url.com/path).
+Spetta agli operatori del cluster decidere quale ingress controller utilizzare per l'esecuzione dell'applicazione web, scegliendo tra una serie di tecnologie possibili come Nginx, Traefik, HAProxy, ecc.
+Grazie a queste caratteristiche, anche quando un'applicazione è composta da più servizi, l'utente può accedervi utilizzando un singolo URL.
+Questo elimina la necessità di numerosi bilanciatori di carico separati a livello di infrastruttura e semplifica la configurazione delle regole del firewall e del bilanciatori di carico per ciascun servizio.
```
Comment (ugho16) [inline content/it/ingress.md:29]:
```suggestion
Questo elimina la necessità di numerosi bilanciatori di carico separati a livello di infrastruttura e semplifica la configurazione delle regole del firewall e dei bilanciatori di carico per ciascun servizio.
```

**Review by ugho16 — APPROVED** [review]

File: content/it/ingress.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
+---
+title: Ingress
+status: Completed
+category: tecnologia
+tags: ["fondamenti"]
+---
+
+Un Ingress è un insieme di regole che aiuta a gestire il traffico Internet proveniente dall'esterno verso un [container](/it/containers/) o un gruppo di container in esecuzione in un [cluster](/it/cluster/).
+È composto da due elementi: la risorsa ingress e dall'ingress controller.
```
Comment (annalisag-spark) [inline content/it/ingress.md:9]:
```suggestion
È composto da due elementi: la risorsa ingress e l'ingress controller.
```

File: content/it/ingress.md [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+L'ingress controller è un server web, eseguito all'interno del cluster, che interpretando le regole definite nella risorsa ingress  
+consente un singolo punto di ingresso attraverso un URL (www.example-url.com) e gestisce l'instradamento effettivo e il bilanciamento in base a diversi percorsi URL (www.example-url.com/path).
+Spetta agli operatori del cluster decidere quale ingress controller utilizzare per l'esecuzione dell'applicazione web, scegliendo tra una serie di tecnologie possibili come Nginx, Traefik, HAProxy, ecc.
+Grazie a queste caratteristiche, anche quando un'applicazione è composta da più servizi, l'utente può accedervi utilizzando un singolo URL.
+Questo elimina la necessità di numerosi bilanciatori di carico separati a livello di infrastruttura e semplifica la configurazione delle regole del firewall e dei bilanciatori di carico per ciascun servizio.
+Centralizzando l'instradamento del traffico e la gestione delle configurazioni, Ingress fornisce una scalabilità ottimizzata, utilizzo efficiente delle risorse, 
+riduzione dei costi e migliora la gestibilità complessiva delle applicazioni in esecuzione in un cluster.
```
Comment (annalisag-spark) [inline content/it/ingress.md:31]: forse c'è un a capo di troppo alla riga 31

in generale deciderei se usare Ingress con la maiuscola sempre o con la minuscola (anche se l'originale stesso è indeciso)

**Review by annalisag-spark — CHANGES_REQUESTED** [review]

**Review by annalisag-spark — APPROVED** [review]

**Pull request merged**